<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bTest">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbImage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>600</width>
      <height>338</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bScreen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Screen</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teOutput">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>891</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNum6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Num 6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leHost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>172.26.21.254</string>
    </property>
    <property name="placeholderText">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePass">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ambartel</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bRestartKodi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart Kodi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCustomCommand">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>569</y>
      <width>891</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bVol0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bUp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDown">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSysType">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>kodi</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bScreenTake">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Screen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leHost</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>leSysType</tabstop>
  <tabstop>bScreen</tabstop>
  <tabstop>teOutput</tabstop>
  <tabstop>bBack</tabstop>
  <tabstop>bNum2</tabstop>
  <tabstop>bNum3</tabstop>
  <tabstop>bNum4</tabstop>
  <tabstop>bNum5</tabstop>
  <tabstop>bNum6</tabstop>
  <tabstop>bTest</tabstop>
  <tabstop>bNum0</tabstop>
  <tabstop>bRestartKodi</tabstop>
  <tabstop>leCustomCommand</tabstop>
  <tabstop>bVol0</tabstop>
  <tabstop>bUp</tabstop>
  <tabstop>bDown</tabstop>
  <tabstop>bNum1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
